jigsawpy/__init__.py

```diff
@@ -50,45 +50,42 @@
 ------------------------------------------------------------
  """
 
-from .msh_t import jigsaw_msh_t
-from .jig_t import jigsaw_jig_t
+from jigsawpy.msh_t import jigsaw_msh_t
+from jigsawpy.jig_t import jigsaw_jig_t
```

Comment:
I don't think relative imports (`.msh_t`) work anymore and they're considered to be bad practice even in python 2 at this point.